maintainers/maintainer-list.nix

```diff
@@ -9699,6 +9699,12 @@
     githubId = 36667224;
     name = "Yingchi Long";
   };
+  IncredibleLaser = {
+    github = "IncredibleLaser";
+    githubId = 45564436;
+    matrix = "@incrediblelaser:tchncs.de";
+    name = "Gereon Schomber";
+  };
```

Comment:
Should be split in a different commit: `maintainers: add IncredibleLaser`.

Reply:
Done

Reply:
maintainers commit should be the first, I went ahead and did it for you